core/src/main/java/nu/yona/server/device/service/DeviceUpdateRequestDto.java

```diff
@@ -11,8 +11,8 @@
 
 public class DeviceUpdateRequestDto
 {
-	final String name;
-	final Optional<String> firebaseInstanceId;
+	final public String name;
+	final public Optional<String> firebaseInstanceId;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)